<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMovieDialog</class>
 <widget class="QDialog" name="AddMovieDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Título</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelYear">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ano:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDuration">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Duração (min):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTitle">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditYear">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDuration">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
